### ISSUE TITLE
As a user of the lib, I'd like to be able to specify local file paths, not only URLs

### ISSUE THREAD

**Issue by sashamitrovich** [frictionlessdata/tableschema-java#64]
# Overview

Not a bug, could be a documentation issue: is it possible to specify local FS path instead of a URL?
---

Please preserve this line to notify @iSnow (lead of this repository)

**Comment by sashamitrovich** [comment]
This would work, right:
Table.fromSource(dataSource, schema, format) 
?

**Comment by iSnow** [comment]
No, that won't work, the "dataSource" parameter is intended to be a JSON string, (Javadoc is there) 

If you want to specify a local FS path, you need to go with `fromSource(File dataSource, File basePath, Schema schema, CSVFormat format)`. Parameger `format`can be null to use the default format.

`dataSource` has to be a relative File path, `basePath` a File pointing to a path to resolve against. The reason is to prevent a class of possible exploits (discussed in "Security" on the Frictionless Data website. Unfortunately, the threat mitigation is over the top if you only want to use TableSchema code, not Datapackages, but I don't have the time to refactor everything to do all security check in the DP code.

The code for `Table` discriminates between Table creation from scratch (the c'tor) and reading an existing Table (the `fromSource()` methods)
